CMakeLists.txt

```diff
@@ -320,6 +321,7 @@ target_compile_options(PythonQt PRIVATE
 target_link_libraries(PythonQt
               ${PYTHON_LIBRARY}
               ${QT_LIBRARIES}
+              Qt5::MultimediaWidgets
```

Comment:
I think Qt5::MultimediaWidgets should be included in ${QT_LIBRARIES} instead of being added manually.